app/assemble/partials/300-javascript/310-writing-and-formatting-javascript.hbs

```diff
@@ -26,9 +26,10 @@ Formatting the language statements and patterns should follow these basics:
 * Conditional statements go on a new line followed by the opening brace.
 * Else/else go on the same line as the brace.
 * Use type strict checks with === as opposed to == whenever possible.
+* Use += and -= instead of ++ and --.
 {{/markdown}}
 {{#code "javascript"}}
-for (var i = 0, len = arr.length; i < len; i++) {
+for (var i = 0, len = arr.length; i < len; i += 1) {
```

Comment:
As a best practice as well as a performance boost, get the length of the array in use prior to a for loop. Acquiring the length of an array every instance of a loop slows performance and creates a delay.

var len = arr.length;
for (var i = 0; i < len; i += 0) {
    // Do something here...
}